python/cuml/umap_dev_tools/test_umap.py

```diff
@@ -453,7 +458,7 @@ def test_simplicial_set_embedding(cu_fuzzy_fixture, params):
         densmap=False,
         densmap_kwds={},
         output_dens=False,
-        output_metric=metric,
+        output_metric="euclidean",
```

Comment:
Why is this now hard-coded to "euclidean"?

Reply:
Some datasets configured for the cosine metric were producing cosine embeddings, but our implementation only supports euclidean embeddings.